log4j-parent/pom.xml

```diff
@@ -576,6 +576,12 @@
         <version>${jna.version}</version>
       </dependency>
 
+      <dependency>
+        <groupId>net.javacrumbs.json-unit</groupId>
+        <artifactId>json-unit-assertj</artifactId>
+        <version>${json-unit.version}</version>
+      </dependency>
+
```

Comment:
Both `json-unit` and `json-unit-assertj` are only used in `log4j-core-test`, shall we move them there instead?

Reply:
Fixed in https://github.com/apache/logging-log4j2/pull/3053/commits/d25e0e6f632fa47f36afaadb7e2af0e4452560bc